slots/utils.py

```diff
@@ -68,12 +68,12 @@ def check_time_slot_for_age_group(member_id, slot_datetime, library):
     member_age = Member.objects.filter(member_id=member_id)[0].age
     if member_age < 16:
         if slot_time := slot_datetime.time():
```

Comment:
not sure if this is a python specific thing, but slot_time can be pulled out of the conditional blocks as it is used down below as well.